validator/src/test/java/com/amazon/aoc/validators/LoadBalancingValidatorTest.java

```diff
@@ -47,26 +48,35 @@ public void testValidationSucceeds() throws Exception {
     validationConfig.setCallingType("http");
     Context context = initContext();
 
-    String mockedTrace =
-        "[{ \"segments\": [{ \"document\": '{ \"metadata\": { \"default\": {\"collector-id\": 3} }, \"subsegments\": [{ \"metadata\": { \"default\": {\"collector-id\": 3} }}, { \"metadata\": { \"default\": {\"collector-id\": 3} }}] }' }] }]";
+    ObjectMapper mapper = new ObjectMapper();
+    JsonNode jsonNode =
+        mapper.readValue(
+            new File(
+                "./src/test/java/com/amazon/aoc/testdata/LoadBalancingValidatorTraceSuccess.json"),
```

Comment:
Can we use `getClass().getResource()` for loading this file? We do similar file loads in other parts of the validator. https://github.com/aws-observability/aws-otel-test-framework/blob/be90b42b6fbe423a59c1d3fc7df208701ff7d70a/validator/src/main/java/com/amazon/aoc/fileconfigs/PredefinedExpectedTemplate.java#L86

Reply:
I've opted to use `File().getAbsolutePath` still as getClass().getResource(path) returns a URL object but we are looking to pass in a String path to `LocalPathExpectedTemplate` for rendering the .mustache file.